internal/action/commands.go

```diff
@@ -68,29 +68,6 @@ func (s *Action) GetCommands() []*cli.Command {
 			Description: "Manages domain aliases. Note: this command might change or go away.",
 			Action:      s.AliasesPrint,
 			Hidden:      true,
-			Subcommands: []*cli.Command{
-				{
-					Name:        "add",
-					Action:      s.AliasesAdd,
-					Usage:       "Add a new alias",
-					ArgsUsage:   "[alias] [domain]",
-					Description: "Adds a new alias",
-				},
-				{
-					Name:        "remove",
-					Action:      s.AliasesRemove,
-					Usage:       "Remove an alias from a domain",
-					ArgsUsage:   "[alias] [domain]",
-					Description: "Remove an alias from a domain",
-				},
-				{
-					Name:        "delete",
-					Action:      s.AliasesDelete,
-					Usage:       "Delete an entire domain",
-					ArgsUsage:   "[alias]",
-					Description: "Delete an entire domain",
-				},
-			},
```

Comment:
Same question. Why remove it?

Reply:
I think this has been discussed / answered above?!